ui/src/components/PollBody/PollBody.jsx

```diff
@@ -0,0 +1,338 @@
+import React, { useState, useEffect } from 'react';
+import AutoSuggest from 'react-autosuggest';
+import Voter from '../Voter/Voter';
+import AnswerOption from '../AnswerOption/AnswerOption';
+import API from '../../API';
+import members from '../../data/voters.json';
+import './PollBody.css';
+
+const PollBody = () => {
+    const [currPage, setCurrPage] = useState(0);
+    const [privacy, setPrivacy] = useState('private');
+    const [value, setValue] = useState('');
+    const [suggestions, setSuggestions] = useState([]);
+
+    const [pollTitle, setPollTitle] = useState("");
+    const [pollDescription, setPollDescription] = useState("");
+    const [pollType, setPollType] = useState("");
+    const [pollExpiration, setPollExpiration] = useState();
+    const [voters, setVoters] = useState([]);
+    const [optionInds, setOptionInds] = useState([1, 2]);
+    const [numForceUpdates, setNumForceUpdates] = useState(0); // BAD PRACTICE ;-;
+    const [options, setOptions] = useState([{optionName: "", description:""}, {optionName:"", description:""}]);
+
+    /* Note to readers: I kind of can't put the pages of the form as their own components? 
+     * Because we need to be able to access all the input values on form submit right :0
+     * But this code does look really messy with that layout especially since useEffect and useState
```

Comment:
I'm pretty sure you can put each page in a different component, provided the JSX at the end
has all of the values. This is because you can pass down boolean props to toggle visibility for
each component and enhance readability of code here. So, for instance, if you had each page
be a different component, this page could still keep all of them here in its JSX, but it'd just
toggle visibility on each one, kind of like how you do already.